uuid-ballerina/uuid.bal

```diff
@@ -0,0 +1,335 @@
+// Copyright (c) 2020 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/java;
+import ballerina/crypto;
+import ballerina/lang.'int as ints;
+import ballerina/stringutils;
+
+# Returns a UUID of type 1 as a string.
+# ```ballerina
+# string uuid1 = uuid:createType1AsString();
+# ```
+#
+# + return - UUID of type 1 as a string
+public isolated function createType1AsString() returns string = @java:Method {
+    name: "generateType1UUID",
+    'class: "org.ballerinalang.stdlib.uuid.nativeimpl.Util"
+} external;
+
+# Returns a UUID of type 1 as a UUID record.
+# ```ballerina
+# Uuid|uuid:Error uuid1 = uuid:createType1AsRecord();
+# ```
+#
+# + return - UUID of type 1 as a UUID record or error
+public isolated function createType1AsRecord() returns Uuid|Error {
+    return check toRecord(createType1AsString());
+}
+
+# Returns a UUID of type 3 as a string.
+# ```ballerina
+# string|uuid:Error uuid3 = uuid:createType3AsString(uuid:NAME_SPACE_DNS, “ballerina.io”);
+# ```
+#
+# + name - name
+# + namespace - namespace
```

Comment:
Wouldn't there be better descriptions of these params, since these are going to the API docs?

Reply:
Updated.